lib/v2/inewsweek/index.js

```diff
@@ -0,0 +1,45 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const iconv = require('iconv-lite');
+
+const rootUrl = 'http://news.inewsweek.cn';
+
+module.exports = async (ctx) => {
+    const channel = ctx.params.channel;
+    const url = `${rootUrl}/${channel}`;
+    const response = await got(url, {
+        responseType: 'buffer',
+    });
+    const $ = cheerio.load(iconv.decode(response.data, 'gbk'));
+    const items = await Promise.all(
+        $('div.grid-item')
+            .toArray()
+            .map((item) => {
+                item = $(item);
+                const href = item.find('a').attr('href');
+                const articleLink = `${rootUrl}${href}`;
+                return ctx.cache.tryGet(articleLink, async () => {
+                    const response = await got(articleLink, {
+                        responseType: 'buffer',
+                    });
+
+                    const $ = cheerio.load(iconv.decode(response.data, 'gbk'));
+                    const fullText = $('div.contenttxt').text();
```

Comment:
```suggestion
                    const fullText = $('div.contenttxt').html();
```
It's better to use `.html()` to preserve formatting and media instead of `.text()` will return a bunch of test without any format